src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -187,28 +189,32 @@ async def chat_completion_over_dataset(
             session.add(experiment)
             await session.flush()
 
+        results = []
+        batch_size = 10
```

Comment:
```suggestion
        batch_size = 3
```

Reply:
after some testing with the rate limiters, we suspect having a large number of tasks competing for relatively few rpm was contributing to coroutine timeouts

Reply:
ok sg. thx